opencog/guile/SchemeSmobTV.cc

```diff
@@ -460,5 +460,32 @@ SCM SchemeSmob::ss_tv_get_value (SCM s)
 	return SCM_EOL;
 }
 
+/**
+ * Return the truth value mean
+ */
+SCM SchemeSmob::ss_tv_get_mean(SCM s)
+{
+	TruthValue *tv = verify_tv(s, "cog-tv-mean");
+	return scm_from_double(tv->getMean());
+}
```

Comment:
Oh, I didn't know about the clunky guile-C++ interface!

I wouldn't mind to have them coded in scheme, expect that given the current bindings I cannot. However, maybe I can keep the existing tv-mean, etc. I removed them to avoid duplicating code, but if that slowing things down that would be a good reason not to remove them.